apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -180,6 +198,8 @@ export function ShowAndTellEntryForm({
 }: ShowAndTellEntryFormProps) {
   const [isSubmitted, setIsSubmitted] = useState(false);
   const [successMessage, setSuccessMessage] = useState<string | null>(null);
+  const [isPreviewOpen, setIsPreviewOpen] = useState(false);
+  const [previewImageUrl, setPreviewImageUrl] = useState<string | null>(null);
```

Comment:
Do we need two states here -- can we not just toggle the modal based on wether there is a URL or not?